### PR TITLE
fix: vs -> vs.

### PR DESCRIPTION
**Pull request by de-oz**
### Description

Append periods to the contraction "vs" as per the [writing style guide](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Writing_style_guide#abbreviations_and_acronyms).

### CODE REVIEW

**Comment by dipikabh** [comment]
Thanks for this pull request, @de-oz. Looking at the changes now.

File: files/en-us/web/api/webvr_api/concepts/index.md [hunk 1]
```diff
@@ -138,7 +138,7 @@ If developers do not take utmost care, VR apps can actually cause their users to
 To avoid bringing on motion sickness in our users (or at least minimize the effects), we need to:
 
 - Always maintain head tracking (this is the most important of all, especially if it occurs in middle of the experience.)
-- Use constant velocity; avoid acceleration or deceleration camera movements (use linear acceleration, and avoid vs easing if you can.)
+- Use constant velocity; avoid acceleration or deceleration camera movements (use linear acceleration, and avoid vs. easing if you can.)
```
Comment (dipikabh) [inline files/en-us/web/api/webvr_api/concepts/index.md:141]:
```suggestion
- Use constant velocity; avoid acceleration or deceleration camera movements (use linear acceleration, and avoid easing if you can.)
```

File: files/en-us/web/api/window/innerheight/index.md [hunk 2]
```diff
@@ -73,7 +73,7 @@ To get the outer height of a window, i.e. the height of the whole browser window
 The following figure shows the difference between `outerHeight` and
 `innerHeight`.
 
-![innerHeight vs outerHeight illustration](firefoxinnervsouterheight2.png)
+![innerHeight vs. outerHeight illustration](firefoxinnervsouterheight2.png)
```
Comment (dipikabh) [inline files/en-us/web/api/window/innerheight/index.md:76]: this is an alt text so the update probably does not matter here so much

File: files/en-us/web/media/formats/image_types/index.md [hunk 3]
```diff
@@ -312,7 +312,7 @@ However it also has disadvantages for some cases, as video and image encoding ha
 The format offers:
 
 - Excellent lossy compression compared to JPG and PNG for visually similar compression levels (e.g. lossy AVIF images are around 50% smaller than JPEG images).
-- Generally AVIF has better compression than WebP — median 50% vs 30% compression for the same JPG set (source: [AVIF WebP Comparison](https://www.ctrl.blog/entry/webp-avif-comparison.html) (CTRL Blog)).
+- Generally AVIF has better compression than WebP — median 50% vs. 30% compression for the same JPG set (source: [AVIF WebP Comparison](https://www.ctrl.blog/entry/webp-avif-comparison.html) (CTRL Blog)).
```
Comment (dipikabh) [inline files/en-us/web/media/formats/image_types/index.md:315]:
```suggestion
- Generally, AVIF has better compression than WebP — median 50% vs. 30% compression for the same JPG set (source: [AVIF WebP Comparison](https://www.ctrl.blog/entry/webp-avif-comparison.html) (CTRL Blog)).
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Just a few nits

**Comment by wbamberg** [comment]
IMO (and this is a very subjective take, I'm happy to go with the consensus) I don't like the use of "versus" here, and like "vs." even less. For example, rather than "Visible vs. logical display surfaces" could we say "Comparing visible and logical display surfaces"? @dipikabh , what do you think?

**Comment by dipikabh** [comment]
> IMO (and this is a very subjective take, I'm happy to go with the consensus) I don't like the use of "versus" here, and like "vs." even less. For example, rather than "Visible vs. logical display surfaces" could we say "Comparing visible and logical display surfaces"?
> 
> @dipikabh , what do you think?

+1. "Comparing visible and logical display surfaces" is definitely better.

File: files/en-us/web/api/window/innerheight/index.md (hunk 2 above)
Comment (dipikabh) [inline files/en-us/web/api/window/innerheight/index.md:76]:
```suggestion
![illustration comparing innerHeight and outerHeight](firefoxinnervsouterheight2.png)
```

File: files/en-us/web/api/window/outerheight/index.md [hunk 4]
```diff
@@ -25,7 +25,7 @@ To get the inner height of a window, i.e. the height of the page being displayed
 
 The following figure shows the difference between `outerHeight` and `innerHeight`.
 
-![innerHeight vs outerHeight illustration](firefoxinnervsouterheight2.png)
+![innerHeight vs. outerHeight illustration](firefoxinnervsouterheight2.png)
```
Comment (dipikabh) [inline files/en-us/web/api/window/outerheight/index.md:28]:
```suggestion
![illustration comparing innerHeight and outerHeight](firefoxinnervsouterheight2.png)
```

File: files/en-us/web/css/revert-layer/index.md [hunk 5]
```diff
@@ -20,7 +20,7 @@ If there is no other cascade layer to revert to for the matching CSS rule, the p
 
 This keyword can be applied to any CSS property, including the CSS shorthand property {{cssxref("all")}}.
 
-## Revert-layer vs revert
+## Revert-layer vs. revert
```
Comment (dipikabh) [inline files/en-us/web/css/revert-layer/index.md:23]:
```suggestion
## Difference between revert-layer and revert
```

File: files/en-us/web/css/revert/index.md [hunk 6]
```diff
@@ -37,7 +37,7 @@ Revert will not affect rules applied to children of an element you reset (but wi
 
 ## Examples
 
-### Revert vs unset
+### Revert vs. unset
```
Comment (dipikabh) [inline files/en-us/web/css/revert/index.md:40]:
```suggestion
### Difference between revert and unset
```

File: files/en-us/web/css/text-rendering/index.md [hunk 7]
```diff
@@ -98,7 +98,7 @@ This demonstrates how `optimizeLegibility` is used by browsers automatically whe
 
 {{ EmbedLiveSample('Automatic_application_of_optimizeLegibility') }}
 
-### optimizeSpeed vs optimizeLegibility
+### optimizeSpeed vs. optimizeLegibility
```
Comment (dipikabh) [inline files/en-us/web/css/text-rendering/index.md:101]:
```suggestion
### Difference between optimizeSpeed and optimizeLegibility
```

File: files/en-us/web/javascript/guide/using_promises/index.md [hunk 8]
```diff
@@ -472,7 +472,7 @@ Promise.resolve()
 console.log(1); // 1, 2, 3, 4
 ```
 
-### Task queues vs microtasks
+### Task queues vs. microtasks
```
Comment (dipikabh) [inline files/en-us/web/javascript/guide/using_promises/index.md:475]:
```suggestion
### Comparing task queues and microtasks
```

File: files/en-us/web/javascript/guide/using_promises/index.md [hunk 9]
```diff
@@ -500,7 +500,7 @@ Promise callback (.then)
 event-loop cycle: Promise (fulfilled) Promise {<fulfilled>}
 ```
 
-For more details, refer to [Tasks vs microtasks](/en-US/docs/Web/API/HTML_DOM_API/Microtask_guide/In_depth#tasks_vs_microtasks).
+For more details, refer to [Tasks vs. microtasks](/en-US/docs/Web/API/HTML_DOM_API/Microtask_guide/In_depth#tasks_vs_microtasks).
```
Comment (dipikabh) [inline files/en-us/web/javascript/guide/using_promises/index.md:503]:
```suggestion
For more details, refer to differences between [tasks and microtasks](/en-US/docs/Web/API/HTML_DOM_API/Microtask_guide/In_depth#Comparing_task_queues_and_microtasks).
```

File: files/en-us/web/javascript/reference/errors/invalid_right_hand_side_instanceof_operand/index.md [hunk 10]
```diff
@@ -39,7 +39,7 @@ i.e. an object which has a `prototype` property and is callable. It can also be
 
 ## Examples
 
-### instanceof vs typeof
+### instanceof vs. typeof
```
Comment (dipikabh) [inline files/en-us/web/javascript/reference/errors/invalid_right_hand_side_instanceof_operand/index.md:42]:
```suggestion
### Comparing instanceof and typeof
```

File: files/en-us/web/javascript/reference/errors/unnamed_function_statement/index.md [hunk 11]
```diff
@@ -34,7 +34,7 @@ You'll need to check how functions are defined and if you need to provide a name
 
 ## Examples
 
-### Statements vs expressions
+### Statements vs. expressions
```
Comment (dipikabh) [inline files/en-us/web/javascript/reference/errors/unnamed_function_statement/index.md:37]:
```suggestion
### Difference between statements and expressions
```

File: files/en-us/web/javascript/reference/global_objects/array/isarray/index.md [hunk 12]
```diff
@@ -68,7 +68,7 @@ Array.isArray(new Uint8Array(32));
 Array.isArray({ __proto__: Array.prototype });
 ```
 
-### instanceof vs Array.isArray()
+### instanceof vs. Array.isArray()
```
Comment (dipikabh) [inline files/en-us/web/javascript/reference/global_objects/array/isarray/index.md:71]:
```suggestion
### Comparing instanceof and Array.isArray()
```

File: files/en-us/web/javascript/reference/global_objects/decodeuri/index.md [hunk 13]
```diff
@@ -64,7 +64,7 @@ decodeURI(
 // "https://developer.mozilla.org/ru/docs/JavaScript_шеллы"
 ```
 
-### decodeURI() vs decodeURIComponent()
+### decodeURI() vs. decodeURIComponent()
```
Comment (dipikabh) [inline files/en-us/web/javascript/reference/global_objects/decodeuri/index.md:67]:
```suggestion
### Comparing decodeURI() and decodeURIComponent()
```

File: files/en-us/web/javascript/reference/global_objects/number/parsefloat/index.md [hunk 14]
```diff
@@ -36,7 +36,7 @@ Or {{jsxref("NaN")}} when the first non-whitespace character cannot be converted
 
 ## Examples
 
-### Number.parseFloat vs parseFloat
+### Number.parseFloat vs. parseFloat
```
Comment (dipikabh) [inline files/en-us/web/javascript/reference/global_objects/number/parsefloat/index.md:39]:
```suggestion
### Comparing Number.parseFloat and parseFloat
```

File: files/en-us/web/javascript/reference/global_objects/number/parseint/index.md [hunk 15]
```diff
@@ -47,7 +47,7 @@ If the `radix` is smaller than `2` or bigger than
 
 ## Examples
 
-### Number.parseInt vs parseInt
+### Number.parseInt vs. parseInt
```
Comment (dipikabh) [inline files/en-us/web/javascript/reference/global_objects/number/parseint/index.md:50]:
```suggestion
### Comparing Number.parseInt and parseInt
```

File: files/en-us/web/javascript/reference/global_objects/reflect/construct/index.md [hunk 16]
```diff
@@ -63,7 +63,7 @@ const obj = new Foo(...args);
 const obj = Reflect.construct(Foo, args);
 ```
 
-### Reflect.construct() vs Object.create()
+### Reflect.construct() vs. Object.create()
```
Comment (dipikabh) [inline files/en-us/web/javascript/reference/global_objects/reflect/construct/index.md:66]:
```suggestion
### Comparing Reflect.construct() and Object.create()
```

File: files/en-us/web/media/formats/audio_concepts/index.md [hunk 17]
```diff
@@ -117,13 +117,13 @@ You can apply a filter that narrows the audio bandwidth, removing any audio freq
...
 
-The most commonly-used compression methods for audio apply the science of **[psychoacoustics](https://en.wikipedia.org/wiki/Psychoacoustics)**. This is the science that studies how humans perceive sound, and what parts of the audio frequencies we hear are most important to how we respond to those sounds, given the context and content of the sound. Factors such as the ability to sense the change in frequency of a sound, the overall range of human hearing versus the frequencies of the audio signal, audio localization, and so forth all can be considered by a codec.
+The most commonly-used compression methods for audio apply the science of **[psychoacoustics](https://en.wikipedia.org/wiki/Psychoacoustics)**. This is the science that studies how humans perceive sound, and what parts of the audio frequencies we hear are most important to how we respond to those sounds, given the context and content of the sound. Factors such as the ability to sense the change in frequency of a sound, the overall range of human hearing vs. the frequencies of the audio signal, audio localization, and so forth all can be considered by a codec.
```
Comment (dipikabh) [inline files/en-us/web/media/formats/audio_concepts/index.md:120]:
```suggestion
The most commonly-used compression methods for audio apply the science of **[psychoacoustics](https://en.wikipedia.org/wiki/Psychoacoustics)**. This is the science that studies how humans perceive sound, and what parts of the audio frequencies we hear are most important to how we respond to those sounds, given the context and content of the sound. Factors such as the ability to sense the change in frequency of a sound, the overall range of human hearing as compared to the frequencies of the audio signal, audio localization, and so forth all can be considered by a codec.
```

File: files/en-us/web/media/formats/audio_concepts/index.md [hunk 18]
```diff
@@ -117,13 +117,13 @@ You can apply a filter that narrows the audio bandwidth, removing any audio freq
...
 
 By using a sound (no pun intended) understanding of psychoacoustics, it's possible to design a compression method that will minimize the compressed size of the audio while maximizing the perceived fidelity of the sound. An algorithm employing psychoacoustics may use any of the techniques mentioned here, and will almost certainly apply others as well.
 
 All of this means there is a fundamental question that has to be asked and answered before choosing a codec: Given the content of the sound, the usage context, and the target audience, is it acceptable to lose some degree of audio fidelity, and if so, how much; or is it necessary that, upon decoding the data, the result be identical to the source audio?
 
-### Lossy vs lossless compression
+### Lossy vs. lossless compression
```
Comment (dipikabh) [inline files/en-us/web/media/formats/audio_concepts/index.md:126]:
```suggestion
### Comparing lossy and lossless compression
```

File: files/en-us/web/performance/rum-vs-synthetic/index.md [hunk 19]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Performance Monitoring: RUM vs synthetic monitoring'
+title: 'Performance Monitoring: RUM vs. synthetic monitoring'
```
Comment (dipikabh) [inline files/en-us/web/performance/rum-vs-synthetic/index.md:2]:
```suggestion
title: 'Performance monitoring'
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Hi @de-oz, based on @wbamberg's valid comment, please consider the suggested changes to avoid the use of 'vs.' where possible and update to a phrase that better conveys the intention of the content.

**Comment by Josh-Cena** [comment]
I'm sorry but I'll push back on this. "A vs. B" is a pretty standard way to say "the difference between A and B". It's the thing most people would actually put into Google search. (Anecdote: just yesterday I was googling "crow vs. raven", literally!)

OTOH, "comparing A and B" is not only long-winded, it's also ambiguous. Who's the person "comparing"? The code author? The JavaScript language? Us the documentation authors? When we say "comparing Dates and strings" on the `==` reference, we are surely not talking about "the difference between Dates and strings", but now we have to tell that from the context. Similarly, changes like "comparing `typeof` and `instanceof`" sounds strictly less natural and may be confused with doing an actual comparison operation.

**Comment by dipikabh** [comment]
> I'm sorry but I'll push back on this. "A vs. B" is a pretty standard way to say "the difference between A and B". It's the thing most people would actually put into Google search. (Anecdote: just yesterday I was googling "crow vs. raven", literally!)

I am okay with "versus" but not the contraction "vs." in headings

> OTOH, "comparing A and B" is not only long-winded—it's also ambiguous. Who's the person "comparing"? The code author? The JavaScript language? Us the documentation authors? \

Usually, the gerund form of the verb in headings is used to demonstrate how to do a task and the subject is the user/reader. So there are headings like "Configuring xyz" or "Monitoring xyz", which means "How to configure xyz" or that this section describes how you, the user/reader, can configure xyz. 
In this case, "Differentiating between" would be more accurate than saying "Comparing" to remove subject ambiguity but I would assume that it is understood that the content/author is providing the comparison between the two items.

> When we say "comparing Dates and strings" on the `==` reference, we are surely not talking about "the difference between Dates and strings"

Sure, in that case, one would not use 'Comparing' so as to avoid the operator meaning confusion.

> changes like "comparing `typeof` and `instanceof`" sound strictly less natural and may be confused with doing an actual comparison operation.

Perhaps, this can stay as "typeof versus instanceof" or be updated to "Difference between typeof and instanceof"

**Comment by dipikabh** [comment]
I am not sure why our Writing Style Guide says not to use the expanded form "versus".

```
"Versus", "vs.", and "v.": The contraction "vs." is preferred.

    Correct: this vs. that
    Incorrect: this v. that
    Incorrect: this versus that
```

Our style guide points to the MS style guide and Chicago Manual of Style.

- As per MS style guide: https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/v/versus-vs:
`In headings, use the abbreviation vs., all lowercase. In text, spell out as versus.`
- No access to Chicago Manual of Style
- On the English language stack exchange: https://english.stackexchange.com/questions/52378/versus-versus-vs-in-writing
- Quoting another source that's not subscribed by our style guide - [Google's developer documentation style guide](https://developers.google.com/style/word-list#letter-v) says:
`Don't use vs. as an abbreviation for versus; instead, use the unabbreviated versus.`

In summary, it is okay to use the contraction 'vs.' in headings (contrary to my writing experience). I'll update the style guide to indicate that it is not incorrect to use the expanded 'versus'.

**Comment by dipikabh** [comment]
> So as per #23412, the resolution is that the use of "vs." in headings in this PR are acceptable, right? :)

Yes :)

Sorry @de-oz for the back and forth on this. Could you revert the changes to your first commit and incorporate my suggestions on that?

File: files/en-us/web/api/window/innerheight/index.md (hunk 2 above)
Comment (de-oz) [reply files/en-us/web/api/window/innerheight/index.md:76]: I agree, but I think it's still better to be consistent. I have no problem with canceling this change and the other similar one if you insist though.

**Review by dipikabh — APPROVED** [review]
Thanks a lot for your contribution and patience on this one, @de-oz 🙂
